src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+
+* {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+/*
+.navBar::after {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+  content: "";
+  width: 40px;
+  height: 40px;
+  background-image: url(images/logo.png);
+  background-size: cover;
+}*/
+
+.navList {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  list-style: none;
+  padding: 0;
+}
+
+.selection .is-active {
+  color: #00acdc;
+  margin-left: 0;
+}
+
+.link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
+  margin-left: 20px;
+  color: #000;
+
+}
+
+.link:hover {
+  color: #00acdc;
+
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  display: block;
+  width: 37px;
+  height: 4px;
+  background-color: #00acdc;
+  top: 56px;
```

Comment:
What if we change header height? Better to use `bottom: 0` in this case

Reply:
Corrected all errors